client/components/domains/registrant-extra-info/fr-form.jsx

```diff
@@ -0,0 +1,319 @@
+	const organizationalFields = [ 'registrantVatId', 'sirenSiret', 'trademarkNumber' ];
+	const conditionalFields = registrantType === 'individual'
+		? individualFields
+		: organizationalFields;
+	return [ 'registrantType', ...conditionalFields ];
+}
+
+class RegistrantExtraInfoForm extends React.PureComponent {
+	static propTypes = {
+		countriesList: PropTypes.object.isRequired,
+		isVisible: PropTypes.bool,
+		onSubmit: PropTypes.func,
+		onStateChanged: PropTypes.func, // Just until we can reduxify the contact details
+	}
+
+	static defaultProps = {
+		countriesList: { data: [] },
+		isVisible: true,
+		isProbablyOrganization: false,
+		values: {},
+		onSubmit: noop,
+		onStateChanged: noop,
+	}
+
+	constructor( props ) {
+		super( props );
+	}
+
+	componentWillMount() {
+		const defaults = {
+			registrantType: this.props.isProbablyOrganization
+				? 'organization' : 'individual',
+			countryOfBirth: 'FR',
+		};
+
+		const values = {
+			...defaults,
+			...this.props.values,
+		};
+
+		// It's possible for the parent to pass in null and still have a form
+		// that the user is happy with, so we pass one state out
+		// immediatly to to make sure there's something valid in the parent
+		this.props.onStateChange( values );
+	}
+
+	handleDobChangeEvent = ( event ) => {
+		// setState() is not syncronous :(
+		const newState = this.newStateFromEvent( event );
+		const { dob_years, dob_months, dob_days } = newState;
+
+		const dateOfBirth = ( dob_years && dob_months && dob_days ) &&
+			[ dob_years, dob_months, dob_days ].join( '-' );
+
+		if ( dateOfBirth ) {
+			debug( 'Setting dateOfBirth to ' + dateOfBirth +
+				( moment( dateOfBirth, 'YYYY-MM-DD' ).isValid() ? '' : ' (invalid)' ) );
+		}
+
+		this.props.onStateChange( {
+			...newState,
+			...( dateOfBirth ? { dateOfBirth } : {} )
+		} );
+	}
+
+	newStateFromEvent( event ) {
+		return {
+			...this.props.values,
+			[ event.target.name ]: event.target.value,
+		};
+	}
+
+	handleChangeEvent = ( event ) => {
+		this.props.onStateChange( this.newStateFromEvent( event ) );
+	}
+
+	// We need a deep comparison to check inside props.values
+	shouldComponentUpdate = ( nextProps ) => {
+		return ! isEqual( this.props, nextProps );
+	}
+
+	render = () => {
+		const translate = this.props.translate;
+		const {
+			registrantType
+		} = this.props.values || {};
+
+		return (
+			<form>
+				<FormFieldset>
+					<FormLegend>
+						{ translate( 'Registering a .FR domain' ) }
+					</FormLegend>
+					{ translate(
+						"Almost done! in order to complete your registration, we're required to collect some additional information:"
+					) }
+				</FormFieldset>
+				<FormFieldset>
+					<FormLegend>
+						{ translate( "Who's this domain for?" ) }
+					</FormLegend>
+					<FormLabel>
+						<FormRadio value="individual"
+							name="registrantType"
+							checked={ 'individual' === registrantType }
+							onChange={ this.handleChangeEvent } />
+						<span>{ translate( 'An individual' ) }</span>
```

Comment:
Hi! I've found a possible matching string that has already been translated **7** times:
`translate( 'Individual' )` *ES Score: 13*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).